app/views/admin/salesforce/actions.html.erb

```diff
@@ -0,0 +1,17 @@
+<% @page_header = "Salesforce Actions" %>
+
+<h3>Refresh Salesforce User Info</h3>
+
+<p>Use the button below to trigger a refresh the cache of information from Salesforce Contact records into Accounts User records.</p>
+
+<p>This operation...</p>
+<ol>
+  <li><b>takes a several minutes to complete</b>,</li>
+  <li>does not write anything to Salesforce, and</li>
+  <li>is called automatically <a href="https://github.com/openstax/accounts/blob/master/config/schedule.rb">the schedule shown here.</a></li>
+</ol>
+
+<%= form_tag(update_users_admin_salesforce_path, method: :put, style: 'margin: 30px 0') do %>
+  <%= submit_tag 'Refresh', class: 'btn btn-primary', disable_with: "Refreshing..." %>
+<% end %>
+
```

Comment:
This page used to be `/app/views/admin/salesforce/show.html.erb` -- renamed it and took out the Salesforce config stuff which is now taken care of in the `openstax_salesforce` engine.